pegasus/sites.v3/code.org/public/ai/pl/101/index.haml

```diff
@@ -0,0 +1,101 @@
+- icon_megaphone = "fa-solid fa-megaphone"
+
+%section.hero-banner-basic.hero-ai-pl-webinar{style: "display: block !important"}
+  .wrapper
+    .text-wrapper.col-75
+      %h1{style: "color: white"}=hoc_s(:ai_pl_101_hero_heading)
+      %p{style: "color: white"}=hoc_s(:ai_pl_101_hero_desc)
+      .hero-ai-pl-webinar__by
+        %span{style: "color: white"}=hoc_s(:ai_pl_101_hero_presented)
+        .hero-ai-pl-webinar__list
+          %img{src: "/images/ai/pl/101/code--w.png", height: "24px", alt: "CODE"}
+          %img{src: "/images/ai/pl/101/khan--w.png", height: "20px", alt: "Khan Academy"}
+          %img{src: "/images/ai/pl/101/iste--w.png", height: "18px", alt: "ISTE"}
+          %img{src: "/images/ai/pl/101/ets--w.png", height: "26px", alt: "ets"}
+
+%section.intro
+  .wrapper
+    %h2=hoc_s(:ai_pl_101_intro_heading)
+    %p=hoc_s(:ai_pl_101_intro_text)
+
+= view :section_divider_line
+
+%section.newsletter-signup
+  .wrapper
+    %h2=hoc_s(:ai_pl_101_join_email_title)
+    %p=hoc_s(:ai_pl_101_pro_join_email_desc)
+    %iframe.col-50{:allowtransparency => "true", :frameborder => "0", :height => "380", :scrolling => "yes", :src => "http://go.pardot.com/l/153401/2023-01-10/pqf2sr", :style => "overflow:hidden; float: none", :type => "text/html", :width => "100%"}
+
+= view :visit_partial_event_logger
+
+= view :section_divider_line
+
+%section
+  .wrapper
+    %h2=hoc_s(:ai_pl_101_explore_title)
+    %p=hoc_s(:ai_pl_101_explore_desc)
+    .action-block__wrapper.action-block__wrapper--two-col
+      .action-block.action-block--explore.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline=hoc_s(:module_label_educator_tool)
+          %h3=hoc_s(:ai_pl_101_explore_title_1)
+          %img{src: "/images/ai/pl/101/explore_1.jpg", alt: ""}
+          %p=hoc_s(:ai_pl_101_explore__desc_1)
+        .content-footer
+          %a{href: "https://blog.khanacademy.org/teacher-khanmigo/?utm_source=code.org&utm_medium=referral&utm_campaign=code.orgwebinar", target: "_blank", class: "link-button",  rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
+            =hoc_s(:ai_pl_101_explore__button_1)
+
+      .action-block.action-block--explore.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline=hoc_s(:module_label_educator_course)
+          %h3=hoc_s(:ai_pl_101_explore_title_2)
+          %img{src: "/images/ai/pl/101/explore_2.jpg", alt: ""}
+          %p=hoc_s(:ai_pl_101_explore__desc_2)
+        .content-footer
+          %a{href: "https://www.iste.org/professional-development/iste-u/artificial-intelligence", target: "_blank", class: "link-button",  rel: "noopener noreferrer", aria:{label: hoc_s(:ai_resources_btn_aria_label_01)}}
+            =hoc_s(:ai_pl_101_explore__button_2)
+
+      .action-block.action-block--explore.action-block--one-col.flex-space-between
+        .content-wrapper
+          %p.overline=hoc_s(:module_label_curriculum)
```

Comment:
I believe this is the string that Dani wanted changed from "Curriculum:" to "Curriculum" (without the ":") so you may need to add a new string to have it not have the ":"